pkg/operation/botanist/addons/nginxingress/nginxingress.go

```diff
@@ -32,7 +33,18 @@ const (
 )
 
 // Values is a set of configuration values for the nginx-ingress component.
-type Values struct{}
+type Values struct {
+	// ImageController is the container image used for nginx-ingress controller.
+	ImageController string
```

Comment:
Nit: Lets keep name little more explanatory
```suggestion
	NginxControllerImage string
```